app/templates/components/footer.twig

```diff
@@ -0,0 +1,4 @@
+<div id="footer">
```

Comment:
We can put anything in the footer - stuff like copyright or credits, etc.
Anything- EXCEPT Javascript- that has its own template!